src/lib389/lib389/cli_idm/account.py

```diff
@@ -221,3 +268,14 @@ def create_parser(subparsers):
     change_pw_parser.add_argument('dn', nargs='?', help='The dn to change the password for')
     change_pw_parser.add_argument('new_password', nargs='?', help='The new password to set')
     change_pw_parser.add_argument('current_password', nargs='?', help='The accounts current password')
+
+    bulk_update_parser = subcommands.add_parser('bulk_update', help='Perform a common operation to a set of entries')
+    bulk_update_parser.set_defaults(func=bulk_update)
+    bulk_update_parser.add_argument('basedn', help="Search base for finding entries, only the children of this DN are processed")
+    bulk_update_parser.add_argument('-f', '--filter', help="Search filter for finding entries, default is '(objectclass=*)'")
+    bulk_update_parser.add_argument('-s', '--scope', choices=['one', 'sub'], help="Search scope (one, sub - default is sub")
+    bulk_update_parser.add_argument('-x', '--stop', action='store_true',
+                                    help="Stop processing updates when an error occurs")
```

Comment:
Probably, a nitpick, but still, I think it makes sense to mention the default here, too **"default=False,"** just for clarity.